apps/web/components/apps/AppList.tsx

```diff
@@ -98,7 +99,6 @@ export const AppList = ({ data, handleDisconnect, variant, listClassName }: AppL
                             updateDefaultAppMutation.mutate({
                               appSlug,
                             });
-                            setBulkUpdateModal(true);
```

Comment:
This is not the right place to add this; it should only be true if the mutation is successful